data/tlg0527/tlg048/greekLit.tlg0527.tlg048.opp-eng2.xml

```diff
@@ -76,7 +76,7 @@ License</p>
 <pubPlace>Cambridge</pubPlace>
 <date>1904</date>
 </imprint>
-<biblScope unit="volume">vol.</biblScope>
+<biblScope unit="volume">Vol.1</biblScope>
```

Comment:
Only use Arabic numbers in this element. So if it is "Volume 1", then just put "1" as the text for the element. The `unit="volume"` attribute shows that it is the volume number that you are talking about.